build-ps/rpm/mysql-5.7-sharedlib-rename.patch

```diff
@@ -11,7 +11,9 @@
  SET_PROPERTY(TARGET client_base PROPERTY EXCLUDE_FROM_ALL TRUE)
 --- client/CMakeLists.txt	2018-08-18 03:29:59.708213642 -0400
 +++ client/CMakeLists.txt	2018-08-18 03:29:59.712547041 -0400
-@@ -36,37 +36,37 @@
+--- client/CMakeLists.txt       2018-12-11 10:52:11.866401807 +0200
++++ /home/korvin/CMakeLists.txt 2018-12-11 12:41:06.216688091 +0200
```

Comment:
"/home/korvin" should be "client" ?

Reply:
fixed in https://github.com/percona/percona-server/pull/2786/commits/f005d9bccbe6be8dc3ba29dac0cc764cd1c98ea1